src/MongoDB/Manager.c

```diff
@@ -477,19 +504,29 @@ zend_object_value php_phongo_manager_create_object(zend_class_entry *class_type
 
 	return retval;
 } /* }}} */
+#endif
 
 bool phongo_add_server_debug(void *item, void *ctx)
 {
+#if PHP_VERSION_ID >= 70000
+        mongoc_server_description_t *server = item;
+        zval                        *retval = ctx;
```

Comment:
Leave the server and retval outside of the version check, they are the same anyway so its nice to make that obvious